app/views/users/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="grid-x grid-margin-x">
+    <%= image_tag(user.photo, :alt => "rss feed", :class => "thumbnail cell medium-6 large-2") %>
```

Comment:
## OPTIOANL

- [ ] Kindly I would suggest you provide another option when the user does not have an image. This blocking the application when the image is not provided.

![image](https://user-images.githubusercontent.com/65068771/155351310-20434d4f-84d8-4519-9701-21c376af2fce.png)